guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2925 @@
+{
```

Comment:
<p>"The <code>Country</code> class can be used to discover the data collections for a country. When working with a particular country, you will find it convenient to get a reference to it using the <code>Country.get()</code> method."</p><ul><li>suggest to combine the two sentences, e.g.</li><li>In order to discover the data collections for a particular country, you may first access the reference variable to it using the <code>country.get()</code>  method, and then fetch the data collections from <code>country.data_collections</code>  property.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>